frontend/src/pages/ErrorsV2/ErrorInstances/ErrorInstances.tsx

```diff
@@ -35,6 +35,7 @@ export interface SearchFormState {
 }
 
 export const ErrorInstances = ({ errorGroup }: Props) => {
+	const [currentSearchEmail, setCurrentSearchEmail] = React.useState('')
```

Comment:
Making a separate state (instead of relying on form) because we only want the highlighted text to change after the form is submitted (instead of as the user is typing).